app/views/layouts/application.html.erb

```diff
@@ -33,25 +33,41 @@
 </head>
 <body>
 
```

Comment:
Go to 7th line in this page. You should see:
```
<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"></script>
   <script>
```
use https. ie:
```
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"></script>
   <script>
```

I think this should solve thumb nail issue. Console is showing following error:
![screenshot from 2015-07-27 10 31 21](https://cloud.githubusercontent.com/assets/7680662/8898976/b23dfec8-344b-11e5-9057-99dbc79f065b.png)

Jquery is necessity for thumbnails of bootstrap.